MorkoUnityProject/Assets/Scripts/ObservableList.cs

```diff
@@ -29,9 +28,6 @@ public void RemoveAt(int index, bool fromEnd = false)
         OnRemove?.Invoke();
         this._data.RemoveAt(fromEnd ? _data.Count - index : index);
     }
-    
-    public int Count()
-    {
-        return this._data.Count;
-    }
-}
\ No newline at end of file
+
+    public int Count => this._data.Count;
+}
```

Comment:
By convention, variables and properties go before functions, move this above add and remove.